packages/morphomax-frontend/vite.config.ts

```diff
@@ -4,33 +4,42 @@ import tailwindcss from '@tailwindcss/vite';
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 
-export default defineConfig({
-  plugins: [
-    react(),
-    tailwindcss(),
-    sentryVitePlugin({
-      org: 'lit-protocol-lw',
-      project: 'vincent-yield-frontend',
-    }),
-    sentryVitePlugin({
-      org: 'lit-protocol-lw',
-      project: 'vincent-yield-frontend',
-    }),
-  ],
+export default defineConfig(({ command }) => {
```

Comment:
Just an FYI -- I guess this is extra extra safe, but the sentryVitePlugin is written to just silently skip uploading sourcemaps if you don't have a valid Sentry env credential... which we shouldn't, anywhere outside of our deployment pipeline, unless we're testing :P I think just removing the explicit `pnpm sentry:sourcemaps` would be enough to solve the problem